conjure-undertow-lib/src/main/java/com/palantir/conjure/java/undertow/lib/Parameters.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.conjure.java.undertow.lib;
+
+import com.google.common.collect.Multimap;
+import com.palantir.logsafe.SafeArg;
+import io.undertow.server.HttpServerExchange;
+import io.undertow.util.AttachmentKey;
+import java.util.Optional;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public final class Parameters {
+
+    private static final Logger log = LoggerFactory.getLogger(Parameters.class);
+
+    public static final AttachmentKey<Multimap<String, String>> SAFE_PATH_PARAMS_ATTACH_KEY =
```

Comment:
I think these should live in the handler

Reply:
Currently, the generator depend on the lib and not on the runtime so we need them here unless we want to change that dependency.

Reply:
ah, my bad.